scripts/apps/archive/views/metadata-view.html

```diff
@@ -38,43 +38,43 @@
         </dl>
 
         <dl>
-            <dt translate>Take key</dt>
+            <dt translate>{{label('anpa_take_key')}}</dt>
             <dd>{{ item.anpa_take_key }}</dd>
         </dl>
         <dl ng-if="item.signal">
             <dt translate>Signal</dt>
             <dd>{{ item.signal }}</dd>
         </dl>
         <dl ng-if="item.anpa_category && item.anpa_category.length">
-            <dt translate>Service</dt>
+            <dt translate>{{label('anpa_category')}}</dt>
             <dd>{{ item.anpa_category | mergeWords:'name' }}</dd>
         </dl>
         <dl ng-repeat="cv in cvs track by cv._id" ng-if="item[cv.schema_field] | mergeWords:'name' : cv._id">
             <dt>{{ cv.display_name | translate }}</dt>
             <dd>{{ getLocaleName(item[cv.schema_field], cv._id) }}</dd>
         </dl>
         <dl ng-if="item.genre  && item.genre.length > 0 && !genreInCvs">
-            <dt translate>Genre</dt>
+            <dt translate>{{label('genre')}}</dt>
             <dd>{{ item.genre | mergeWords:'name'}}</dd>
         </dl>
         <dl ng-if="item.place && item.place.length > 0 && !placeInCvs">
-            <dt translate>Place</dt>
+            <dt translate>{{label('place')}}</dt>
             <dd>{{ item.place | mergeWords:'name'}}</dd>
         </dl>
         <dl ng-if="item.ednote">
-            <dt translate>Editorial Note</dt>
+            <dt translate>{{label('ednote')}}</dt>
             <dd>{{ item.ednote }}</dd>
         </dl>
         <dl>
-            <dt translate>Dateline</dt>
+            <dt translate>{{label('dateline')}}</dt>
             <dd>{{ item.dateline.text }}</dd>
         </dl>
         <dl>
-            <dt translate>Byline</dt>
+            <dt translate>{{label('byline')}}</dt>
             <dd>{{ item.byline }}</dd>
         </dl>
         <dl>
-            <dt translate>Sign-Off</dt>
+            <dt translate>{{label('sign_off')}}</dt>
```

Comment:
I suppose we don't need that translate here anymore, do we?

Reply:
Yep, all translate attributes need to be removed where `label` function is used.